fastapi_sqla/__init__.py

```diff
@@ -243,13 +245,16 @@ def _paginate(
     total_items: int,
     offset: int,
     limit: int,
+    *,
+    scalars: bool = True,
```

Comment:
Should this be singular `scalar`? (I'm not sure, I'm asking you @hadrien )

Reply:
JK, I got confused between the old `.scalar()` from the legacy queries and `Results.scalars()`

Reply:
it is plural as it return multiple scalar.